site/content/tutorial/13-advanced-styling/03-style-directive/text.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Inline styles and the style directive
+---
+
+Being able to set css properties dynamically is nice, however this can get unwieldly if you have to write a long string, and mistakes like missing any of the semicolons could make the whole string invalid. Therefore, Svelte provides a nicer way to write inline styles with the style directive.
+
+Change the style attribute of the paragraph to the following:
+
+```html
+<p 
+	style:color 
+	style:--opacity="{bgOpacity}"
+>
+```
+
+Style directives share a few qualities with class directives. You can use a shorthand when the name of the property and the variable are the same. So `style:color="{color}"` can be written as just `style:color`. And just like class directives, the directives will take precedence if you set the same property through a style attribute.
```

Comment:
my english is not that good, but i think we can break this long paragraph into shorter ones

```suggestion
The style directive shares a few qualities with the class directive. You can use a shorthand when the name of the property and the variable are the same. So `style:color="{color}"` can be written as just `style:color`.

Similar to the class directive, the style directive will take precedence if you try to set the same property through a style attribute.
```